tests/Platforms/MariaDBPlatformTest.php

```diff
@@ -74,4 +74,25 @@ public function testIgnoresDifferenceInDefaultValuesForUnsupportedColumnTypes():
     {
         self::markTestSkipped('MariaDB supports default values for BLOB and TEXT columns');
     }
+
+    protected function getGenerateForeignKeySql(): string
+    {
+        return 'ALTER TABLE test ADD FOREIGN KEY (`fk_name_id`) REFERENCES `other_table` (`id`)'
+            . ' ON UPDATE NO ACTION ON DELETE NO ACTION';
```

Comment:
Unlike 4.x where the referential action on a constraint is optional and the corresponding SQL is generated only if it's specified, in 5.0 the default action is `NO ACTION` (per the standard). MariaDB defaults to `RESTRICT`, so we have to render `NO ACTION` in DDL explicitly. This is not a breaking change because `NO ACTION` and `RESTRICT` are synonyms on MariaDB. Why didn't they default to `NO ACTION`? 🤷‍♂️